dev-python/gst-python/gst-python-1.24.10.ebuild

```diff
@@ -20,6 +20,7 @@ RDEPEND="${PYTHON_DEPS}
 	>=media-libs/gstreamer-${PV}:1.0[introspection]
 	>=media-libs/gst-plugins-base-${PV}:1.0[introspection]
 	>=dev-python/pygobject-3.8:3[${PYTHON_USEDEP}]
+	<dev-python/pygobject-3.52:3
```

Comment:
This requires a revision bump to propagate the runtime dependency changes.

Reply:
Sorry, when I said that this requires a revision bump. I meant that the ebuilds should be renamed `-rX -> -rX+1`, not that they should be copied to `-rX+1`. 

We should also remove the old versions otherwise portage might incorrectly try to install the previous revisions to satisfy the dependencies, which would then fail precisely on the problem we are trying to solve here.